### PR TITLE
Further changes adding more guidance for developers on apks

### PR DESCRIPTION
**Pull request by PaulKinlan**

### CODE REVIEW

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 1]
```diff
@@ -141,42 +141,82 @@ Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
 your Service Worker. It is possible for one installed Web Application to be
 controlled by several different Service Workers.
 
-## Navigating outside of my progressive web app
+## Navigating outside of your progressive web app
+
+It is important for users to feel like they are using an app-like experience
+when using your progressive web app. This means that we need to handle
+navigations sensetively. If the user navigates to another origin then your
+progressive web app should not act like a container, instead a new context will
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: 'Instead' should be the start of a new sentence.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 2]
```diff
@@ -141,42 +141,82 @@ Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
...
+scope will cause a new web context to be opened for the user. This allows you to
+have multiple progressive web apps per origin.
+
+## Effectively managing your app's scope
+
+The `scope` attribute in your web app manifest is one of the most important
+things that you will need to do to ensure your app functions as expected.
+
+The `scope` controls the url structure that encompasses all the entry and exit
+points in your web app. Your `start_url` needs to reside in the your `scope`.
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: This is different from service worker scope, isn't it? You should address this somewhere.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 3]
```diff
@@ -141,42 +141,82 @@ Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
...
-sites show up on the home screen, app drawer and throughout the Android 
-System-UI as a user would expect. Permissions are handled differently, by 
-default your app can only have the same permissions surface as Chrome would 
-normally have when installed - you can't ask for Camera access at install time 
-for example. This means that as developer you must request permission for 
-sensitive API's such as Camera and Microphone access, notifications etc at 
-runtime as you would for any normal web site and the Chrome runtime will prompt 
+By Installing your Progressive Web App it now becomes part of the system. Added
+sites show up on the home screen, app drawer and throughout the Android
+System-UI as a user would expect. Permissions are handled differently, by
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: 'System UI' not System-UI. Check throughout.

Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: 'by default' is the start of a new sentence.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 4]
```diff
@@ -141,42 +141,82 @@ Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
...
+default your app can only have the same permissions surface as Chrome would
+normally have when installed - you can't ask for Camera access at install time
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: camera should be lowercase.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 5]
```diff
@@ -141,42 +141,82 @@ Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
...
+for example. This means that as developer you must request permission for
+sensitive API's such as Camera and Microphone access, notifications etc at
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: 'camera and microphone' not 'Camera and Microphone'. Please check throughout.

Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: etc. needs a period.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 6]
```diff
@@ -141,42 +141,82 @@ Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
...
+runtime as you would for any normal web site and the Chrome runtime will prompt
 you for access.
 
-Android normally gives instant access notifications, Installed Progressive Web 
-Apps do not have this permission granted by default and your user must 
+Android normally gives instant access notifications, Installed Progressive Web
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: 'Installed progressive web apps' is the start of a new sentence.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 7]
```diff
@@ -141,42 +141,82 @@ Note: The `scope` in the Web App Manifest, is not the same as the `scope` of
...
 explicitly opt-in to receiving notifications
 
 ## Storage and app state
 
-When the user adds your Progressive Web App to their system Chrome will use the 
-same profile and will not segregate the data. This means your service worker 
-will already be installed, your cookies still active any client-side storage 
+When the user adds your Progressive Web App to their system Chrome will use the
+same profile and will not segregate the data. This means your service worker
+will already be installed, your cookies still active any client-side storage
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: 'active' to 'activate'

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 8]
```diff
@@ -190,8 +230,8 @@ banner](/web/fundamentals/engage-and-retain/app-install-banners/).
 
 We recommend using [Lighthouse](/web/tools/lighthouse/) to audit your PWA.
 
-Note though that there is no engagement threshold for improved add to home 
-screen from the menu.
+Note: there is no engagement threshold for improved add to home screen from the
+menu, it is only used to determine when to prompt users to install directly.
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: 'it is only' is the start of a new sentence.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md [hunk 9]
```diff
@@ -259,13 +304,13 @@ No, the site opens in the version of Chrome the user added the site from.
 
 ### If I update my site's service worker, will this update automatically in the background even without the user visiting the site?
 
-No. The update to the SW will be processed the next time that the user visits 
+No. The update to the SW will be processed the next time that the user visits
```
Comment (jpmedley) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: spell out service worker.

File: src/content/en/updates/2017/02/improved-add-to-home-screen.md (hunk 2 above)
Comment (PaulKinlan) [inline src/content/en/updates/2017/02/improved-add-to-home-screen.md]: done.

**Comment by jpmedley** [comment]
@PaulKinlan This is marked 'do not merge'. Do you need help to move this forward?

**Comment by PaulKinlan** [comment]
@jpmedley not right now, I am waiting on some product decisions.

**Comment by PaulKinlan** [comment]
@petele this is ready from my side. More updates will follow.

**Comment by PaulKinlan** [comment]
Conflicts resolved. @petele @jpmedley

**Comment by petele** [comment]
will push live Tuesday AM

**Pull request merged**